private/tools/java/rules/jvm/external/maven/MavenPublisher.java

```diff
@@ -289,6 +303,14 @@ private static Callable<Void> s3upload(String targetUrl, Path toUpload) {
     };
   }
 
+  private static Callable<Void> arUpload(String targetUrl, Path toUpload) {
+    return () -> {
+      Credentials cred = GoogleCredentials.getApplicationDefault();
+      String url = "https://" + targetUrl.substring(19);
```

Comment:
Presumably, `19` is the length of the scheme?

Reply:
yes, it's `len("artifactregistry://")`